pd/client.go

```diff
@@ -61,13 +62,21 @@ var (
 )
 
 type client struct {
-	url        string
-	tag        string
-	clusterID  uint64
-	clientConn *grpc.ClientConn
+	url       []string
+	tag       string
+	clusterID uint64
+
+	connMu        sync.RWMutex
+	clientConn    *grpc.ClientConn
```

Comment:
Can we maintain all the connections for each URL like the way https://github.com/pingcap/pd/blob/master/client/client.go do?